backend/estimate-service/src/main/java/org/egov/works/web/models/SaveEstimate.java

```diff
@@ -0,0 +1,28 @@
+package org.egov.works.web.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+import org.springframework.validation.annotation.Validated;
+
+/**
+ * SaveEstimate
+ */
+@Validated
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2022-07-27T14:01:35.043+05:30")
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class SaveEstimate {
```

Comment:
Why this separate class? We don't need to create separate classes for kafka topic objects. 
Same applies for update estimate.

Reply:
These were mentioned in the swagger doc [here](https://app.swaggerhub.com/apis/eGovTrial/Estimate-Service/1.0.0#/save-estimate.) hence while generating the project these objects got created.